2019/index.html

```diff
@@ -0,0 +1,72 @@
+---
+---
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css?family=Gugi|Roboto+Condensed" rel="stylesheet">
+    <link rel="stylesheet" href="css/base.css">
+    <title>VimConf 2019</title>
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <nav>
+        </nav>
+      </header>
+      <div class="row justify-content-center">
+        <div class="jumbotron">
+          <div class="row">
+            <div class="logo col-sm-4 justify-content-center">
+              <img src="images/vimlogo-564x564.png" alt="Vim logo" class="img-fluid"/>
+            </div>
+            <div class="info col-sm-8 justify-content-center align-self-center">
+              <h1 class="text-left">
+                VimConf 2019<br />
+              </h1>
+              <h3 class="text-left">
+                <span class="font-italic">November 03, 2019</span>
+              </h3>
+              <p class="lead text-left"> VimConf is an international conference about Vim, the editor.</p>
+            </div>
+          </div>
+          <hr class="my-4">
+          <div class="details text-center">
+            <div class="location ">
+              <h2 class="font-italic">
+                @Akiba Hall, Akihabara, Tokyo
+              </h2>
+            </div>
+            <div class="contact">
+              <ul>
+                <li>
+                  <i class="fas fa-envelope"></i>
+                  <a href="mailto:vimconf@gmail.com?subject=feedback">vimconf@gmail.com</a>
+                </li>
+                <li>
+                  <i class="fab fa-twitter"></i>
+                  <a href="https://twitter.com/vim_jp" target="_blank">https://twitter.com/vim_jp</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <hr class="my-4">
+          <div class="sponsorship text-center">
+            <p>
+            Want to be a sponsor?<br />
+            <strong><a href="https://forms.gle/dWm6TUi3yfDr5UyL7" target="_blank">Get in touch!</a></strong>
```

Comment:
```suggestion
            <strong><a href="https://forms.gle/KzpBDA5dbeYhagyE8" target="_blank">Get in touch!</a></strong>
```

スポンサー応募フォームこちらでお願いします！